### DISCUSSION TITLE
Migration cli, can we force 'yes' on data loss warnings for the sake of Prod builds?

### DISCUSSION THREAD

**Discussion by jaeming** [prisma/prisma#2938] [Q&A] [answered]
When you have a migration that will incur a data loss, you are presented with a confirmation prompt. For example, when running `prisma migrate up`, you may see something like this:
```

⚠️  There will be data loss:

  • You are about to drop the table `Roles`, which is not empty (5 rows).

✔ Are you sure you want to apply this change? … yes
```
This is problematic for Production builds as it can cause the build job to stall out.
I don't see anything in the CLI documentation to force a 'yes' answer to this prompt. The closest thing I see is `--create-db` but that is specific for the db creation prompt. It'd be nice if there was something similar for data loss operations perhaps? Or maybe, a `-y` (yes all) type of flag?

Or is there something undocumented that I'm missing?

**Comment by ryands17** [comment, marked as answer]
Hey @jaeming :wave: 
You can use the `--auto-approve` argument to bypass this during builds.

`prisma migrate up --auto-approve --experimental`

**Reply by jaeming** [reply]
@ryands17 , thanks. That is exactly what I was looking for.

**Comment by alyssa-dannielle** [comment]
`npx prisma migrate reset -f` is what worked for me for our pre-commit hook

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
